src/pages/Tutorials/Markdown/Universal-Deploy.md

```diff
@@ -0,0 +1,149 @@
+
+```python
+pip install prefect -U
+conda install prefect -c conda-forge
+```
+
+# Run a Flow Using Prefect Core
+
+```python
+import prefect
+from prefect import task, Flow
+
+@task
+def hello_task():
+logger = prefect.context.get("logger")
+logger.info("Hello world!")
+
+flow = Flow("hello-flow", tasks=[hello_task])
+
+flow.run()
+```
+
+Paste the code above into an interactive Python REPL session. You should see the following logs after running `flow.run()` :
+
+```
+[2020-01-08 23:49:00,239] INFO - prefect.FlowRunner | Beginning Flow run for 'hello-flow'
+[2020-01-08 23:49:00,242] INFO - prefect.FlowRunner | Starting flow run.
+[2020-01-08 23:49:00,249] INFO - prefect.TaskRunner | Task 'hello_task': Starting task run...
+[2020-01-08 23:49:00,249] INFO - prefect.Task: hello_task | Hello world!
+[2020-01-08 23:49:00,251] INFO - prefect.TaskRunner | Task 'hello_task': finished task run for task with final state: 'Success'
+[2020-01-08 23:49:00,252] INFO - prefect.FlowRunner | Flow run SUCCESS: all reference tasks succeeded
+```
+
+If you're running into issues, check that your Python environment is properly set up to run Prefect. Refer to the [Prefect Core Installation](https://docs.prefect.io/core/getting_started/installation.html) documentation for further details.
+
+# Log into Prefect Cloud from the CLI
+
+To authenticate, you'll need to create a [Personal Access Token](/user/tokens) and configure it with the [Prefect Command Line Interface](https://docs.prefect.io/orchestration/concepts/cli.html#cli).
+
+- In the hamburger menu in the top left corner go to **User -> Personal Access Tokens -> Create A Token**.
+- Copy the created token
+- Configure the CLI to use the access token by running
+
+```bash
+prefect auth login -t <COPIED_TOKEN>
+```
+
+# Create an agent token
+
+Our next goal is to generate an agent token.
+
+[Agents](https://docs.prefect.io/orchestration/agents/overview.html) are responsible for communicating with Prefect Cloud and submitting your flows for execution. Your agents will need tokens called RUNNER tokens to authenticate with Prefect Cloud. There is no limit to the number of agent tokens you can create. You can create one using the CLI:
+
+```bash
+prefect auth create-token -n my-runner-token -s RUNNER
+```
+
+You'll need this token later in the tutorial. You can save it in your local configuration either as an environment variable or by storing it in `~/.prefect/config.toml`:
+
+```bash
+export PREFECT__CLOUD__AGENT__AUTH_TOKEN=<COPIED_RUNNER_TOKEN>
+```
+
+# Creating a project
+
+Projects are used to organize flows that have been deployed to Prefect Cloud.
+
+Every time you deploy a flow, you will need to specify a project to deploy into. There are no limits on the number of projects you can have, and you can always delete projects later. You can read more about interacting with projects [here](https://docs.prefect.io/cloud/concepts/projects.html).
+
+**UI**
```

Comment:
Okay, I changed the bolds to headings but in the toc, it won't render as a child of a section. Right now, I can't figure out a way to group all the smaller headings and make that a child of all the h1
<img width="280" alt="Screen Shot 2020-11-19 at 1 38 28 PM" src="https://user-images.githubusercontent.com/40726391/99727275-92a42e80-2a6c-11eb-8f2d-aa51a22b6fb8.png">

Reply:
Hmm... Good point.  Maybe we try again with the bold but make sure the spacing is correct.  So it looks more like this:
<img width="653" alt="Screen Shot 2020-11-19 at 1 14 19 PM" src="https://user-images.githubusercontent.com/40272060/99832466-90a2a400-2b2e-11eb-9212-fde5d6f21d65.png">
Rather than this (which is what I was seeing before):
<img width="651" alt="Screen Shot 2020-11-20 at 12 51 27 PM" src="https://user-images.githubusercontent.com/40272060/99832813-1e7e8f00-2b2f-11eb-9157-b949dda2c3bc.png">

Reply:
I asked Nicholas about this and he said it was okay not to have the smaller headings nested in the toc but I did change all the bolds to h4 
<img width="1098" alt="heading" src="https://user-images.githubusercontent.com/40726391/99834758-8b3a5f00-2b18-11eb-8001-83951cbeb559.png">